assets/js/components/ReportError.test.js

```diff
@@ -97,4 +98,38 @@ describe( 'ReportError', () => {
 			'Insufficient permissions in Test Module'
 		);
 	} );
+
+	it( 'renders the `Retry` button if the error has `selectorData` property', () => {
```

Comment:
This test spec isn't quite accurate, as we check for attributes _on_ the `selectorData` object, not just for the presence of `selectorData` at all. Arguably a test that has `selectorData: {}` (shouldn't ever happen, but just in case) should _not_ show the retry button and would be worth having.

This test should be named a bit more explicitly.